contracts/CRC.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.5.0;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721Full.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721Pausable.sol";
+
+contract CRC is Initializable, ERC721, ERC721Enumerable, ERC721Metadata, ERC721Pausable {
```

Comment:
an example of a dead-simple rewrite of the CRC contract. Verification data can be included in the `tokenURI` param of the `mintAndRetire` function. 

From 1,000+ custom lines of solidity to just 50.